lib/Settings.js

```diff
@@ -41,6 +43,15 @@ Settings.loadRepositoriesSettings = function (configPath) {
         if (!defined(configJson[name]['gitHubToken'])) {
             return Promise.reject('repository ' + name + ' must have a `gitHubToken`');
         }
+        thirdPartyFolders = configJson[name]['thirdPartyFolders'];
+        if (defined(thirdPartyFolders)) {
+            for (var j = 0; j < thirdPartyFolders.length; j++) {
+                var folder = thirdPartyFolders[j];
+                if (folder.startsWith('/')) {
```

Comment:
Does this have to worry about trailing slashes as well?  I i.e. `ThirdParty/` vs `ThirdParty`